src/Jank/Scorepads/Player.tsx

```diff
@@ -0,0 +1,18 @@
+import { Text, Picture } from './ui';
+import defaultPicture from '../../UserManagement/user_default.png';
+
+type PlayerProps = {
+  name: string;
+  picture: string;
+}
+
+const Player = ({ name, picture = defaultPicture }: PlayerProps) => (
+  <>
+    <Picture src={`/picture/${picture}`} alt="PIC" />
```

Comment:
The `alt` attribute is usually used to provide an alternative text in case the image could not be displayed (e.g. because it failed to download, or because a screen reader is used for accessibility reasons, or when used in mails that don't render HTML and so on).

So it should be descriptive of the image being displayed instead of being generic text. Having generic text is pretty much the same as not having any `alt` text ;) I guess in this case it was required by the linter, but in general I'd rather disable the rule instead of forcing dummy values here. Or, preferably, you could use an actual `alt` text, like `Picture of ${name}` or `${name}'s profile picture` or something. So even if the image could not be displayed, it would be clear what would usually be shown in its place.